DOCS/man/mpv.rst

```diff
@@ -711,22 +711,25 @@ or at runtime with the ``apply-profile <name>`` command.
         # a profile that can be enabled with --profile=big-cache
         [big-cache]
         cache=yes
-        demuxer-max-bytes=123400KiB
+        demuxer-max-bytes=512MiB
         demuxer-readahead-secs=20
 
-        [slow]
-        profile-desc="some profile name"
-        # reference a builtin profile
-        profile=high-quality
+        [network]
+        profile-desc="profile for content over network"
+        profile-cond=demuxer_via_network
+        force-window=immediate
+        # you can also include other profiles
+        profile=big-cache
 
-        [fast]
-        vo=vdpau
+        [reduce-judder]
+        video-sync=display-resample
+        interpolation=yes
 
         # using a profile again extends it
-        [slow]
-        framedrop=no
-        # you can also include other profiles
-        profile=big-cache
+        [network]
+        demuxer-max-back-bytes=512MiB
```

Comment:
no-op since network includes big-cache
different example you could use: `network-timeout=30`

Reply:
big cache has `demuxer-max-bytes`, this sets `demuxer-max-back-bytes`. I don't think this is a no-op?

Reply:
oh yeah, my mistake